scripts/coverage-targeted.sh

```
@@ -0,0 +1,181 @@
+  exit 5
+fi
+JACOCO_AGENT_JAR="${HOME_DIR}/.m2/repository/org/jacoco/org.jacoco.agent/${JACOCO_VERSION}/org.jacoco.agent-${JACOCO_VERSION}-runtime.jar"
+
+rm -f "$EXEC_FILE"
+
+if [[ ! -f "$JACOCO_AGENT_JAR" ]]; then
+  echo "JaCoCo agent not found; downloading ${JACOCO_VERSION}..." >&2
+  (
+    cd "$ROOT_DIR"
+    ./mvnw -q org.apache.maven.plugins:maven-dependency-plugin:3.8.1:get \
+      -Dartifact=org.jacoco:org.jacoco.agent:${JACOCO_VERSION}:jar:runtime
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)